WeatherService.Tests/ParsersTest/WeatherParserFactoryShould.cs

```diff
@@ -0,0 +1,36 @@
+﻿using FluentAssertions;
+using System;
+using System.IO;
+using WeatherMonitoringReportingService;
+using WeatherMonitoringReportingService.DataFiles;
+using WeatherMonitoringReportingService.Parsers;
+using Xunit;
+
+namespace WeatherService.Tests
+{
+    public class WeatherParserFactoryShould
+    {
+        [Fact]  
+        public void returnJSONParserObjectWhenJsonParserNeeded()
```

Comment:
read more about [naming conventions for unit test please ](https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices)